specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/job.json

```diff
@@ -0,0 +1,931 @@
+                    "type": "string",
+                    "description": "The job configuration file."
+                },
+                "jobId": {
+                    "type": "string",
+                    "description": "The full ID of the job."
+                },
+                "jobID": {
+                    "$ref": "#/definitions/JobID",
+                    "description": "The ID of the job."
+                },
+                "jobName": {
+                    "type": "string",
+                    "description": "The user-specified job name."
+                },
+                "jobPriority": {
+                    "type": "string",
+                    "description": "The priority of the job."
+                },
+                "mapProgress": {
+                    "type": "number",
+                    "format": "double",
+                    "x-nullable": false,
+                    "description": "The progress made on the maps."
+                },
+                "neededMem": {
+                    "type": "integer",
+                    "format": "int64",
+                    "x-nullable": false,
+                    "description": "The amount of memory needed for the job."
+                },
+                "numReservedSlots": {
+                    "type": "integer",
+                    "format": "int32",
+                    "x-nullable": false,
+                    "description": "The number of slots reserved."
+                },
+                "numUsedSlots": {
+                    "type": "integer",
+                    "format": "int32",
+                    "x-nullable": false,
+                    "description": "The number of slots used for the job."
+                },
+                "priority": {
+                    "type": "string",
+                    "description": "The priority of the job."
+                },
+                "queue": {
+                    "type": "string",
+                    "description": "The job queue name."
+                },
+                "reduceProgress": {
+                    "type": "number",
+                    "format": "double",
+                    "x-nullable": false,
+                    "description": "The progress made on the reduces."
+                },
+                "reservedMem": {
+                    "type": "integer",
+                    "format": "int64",
+                    "x-nullable": false,
+                    "description": "The amount of memory reserved for the job."
+                },
+                "retired": {
+                    "type": "boolean",
+                    "x-nullable": false,
+                    "description": "Whether or not the job has been retired. "
+                },
+                "runState": {
+                    "type": "integer",
+                    "format": "int32",
+                    "x-nullable": false,
+                    "description": "The current state of the job."

```

Comment:
Is this an enum? Is there a set of predefined value? What would runState == 4711 mean?